plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -43,7 +43,7 @@
     <consolePatternMatchListener
       class="com.google.cloud.tools.eclipse.ui.util.console.PatternToHyperlinkConverter"
       id="com.google.cloud.tools.eclipse.appengine.deploy.consolePatternMatchListener"
-      regex="https://\S+\.appspot\.com">
+      regex="https://[a-zA-Z][a-zA-Z0-9-.]*[a-zA-Z0-9]?\.appspot\.com">
```

Comment:
That still matches `abc-`.  Since a Project ID ≥ 6 characters you can leave off the trailing `?`